moveit_core/robot_state/include/moveit/robot_state/robot_state.h

```diff
@@ -1647,7 +1659,9 @@ class RobotState
    * the body positions will get corrupted.  You need to make a fresh
    * copy of the AttachedBody object for each RobotState you attach it
    * to.*/
-  void attachBody(AttachedBody* attached_body);
+  [[deprecated("Deprecated use the overload requiring an unique_ptr instead")]] void
```

Comment:
```suggestion
  [[deprecated("Deprecated use the overload requiring a unique_ptr instead")]] void
```
https://preply.com/en/question/which-is-correct-a-unique-or-an-unique-49748  English is such a lovely language :wink:

Reply:
Nice catch! Changed it to @rhaschke's suggestion.